ldap/servers/slapd/back-ldbm/ldbm_modrdn.c

```diff
@@ -566,8 +569,8 @@ ldbm_back_modrdn(Slapi_PBlock *pb)
              * instead.  Otherwise, a ModRDN operation will potentially change an
              * entry's entire DN (at least with respect to case and spacing).
              */
-            slapi_sdn_copy(slapi_entry_get_sdn_const(e->ep_entry), sdn);
-            slapi_pblock_set(pb, SLAPI_MODRDN_TARGET_SDN, sdn);
+            //slapi_sdn_copy(slapi_entry_get_sdn_const(e->ep_entry), sdn); // MARK This resets new superior case
+            //slapi_pblock_set(pb, SLAPI_MODRDN_TARGET_SDN, sdn);
```

Comment:
@progier389 @tbordaz  Ok so please look at this code and the long comment about changing the case.  I am sort of breaking what the code was originally trying to do.  In my case I want the new superior components (the ones that are not changing) to match the case of the original entry.  So resetting the superior would break what I am trying to do (preserver the case of the entire DN where possible).  Anyway I am hesitant to change the behavior, but I need to in order to conform with what I am trying to do.   Perhaps I am not seeing the potential risk of changing this behavior?  

Let me know what you guys think about this code/comment above...